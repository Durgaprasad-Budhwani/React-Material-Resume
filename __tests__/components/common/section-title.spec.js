/**
 * Created by durgaprasad on 10/9/17.
 */
/* global expect,it */
import React from 'react';
import SectionTitle from '../../../src/components/common/section-title';
import renderer from 'react-test-renderer';

it('renders section title', () => {
  const tree = renderer.create(<SectionTitle title="Hello" />).toJSON();
  expect(tree).toMatchSnapshot();
});