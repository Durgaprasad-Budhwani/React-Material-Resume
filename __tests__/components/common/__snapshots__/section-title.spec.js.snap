// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders section title 1`] = `
<span
  className="SectionTitle-sectionTitle-1"
>
  Hello
</span>
`;
